assets/js/phoenix_live_view.js

```diff
@@ -968,6 +970,12 @@ export let DOM = {
 
   isFormInput(el){ return /^(?:input|select|textarea|button)$/i.test(el.tagName) },
 
+  syncAttrsToProps(el){
+    if(el instanceof HTMLInputElement && el.type.toLocaleLowerCase() === "checkbox"){
```

Comment:
So radio buttons also have a `.checked` property. Maybe we need:

`const CHECKABLE_INPUTS = ["checkbox", "radio"]`

```suggestion
    if(el instanceof HTMLInputElement && CHECKABLE_INPUTS.indexcOf(el.type.toLocaleLowerCase()) >= 0){
```